build/webpack.styles.js

```diff
@@ -1,35 +1,61 @@
 import ExtractTextPlugin from 'extract-text-webpack-plugin'
-// import postcssPlugins from './webpack.postcss'
 
-const loaderOptions = [
-  { loader: 'css-loader', query: { modules: true, importLoaders: 1 } },
-  { loader: 'postcss-loader'
-    // pending new postcss-loader release - then remove loaderOptionsPlugin
-    // options: {
-    //   plugins: function () {
-    //     return postcssPlugins
-    //   }
-    // }
+const style = 'style-loader'
+const css = 'css-loader'
+const postcss = 'postcss-loader'
+const sass = 'sass-loader'
+
+const cssOptions = [{
+  loader: css,
+  query: {
+    modules: true,
+    importLoaders: 1
   }
+}, {
+  loader: postcss
+}]
+
+const sassOptions = [
+  { loader: css, query: { modules: false, importLoaders: 1 } },
+  { loader: postcss },
+  { loader: sass, query: { outputStyle: 'compressed', includePaths: /node_modules/ } }
 ]
 
 const styleLoaders = {
-  development: {
-    test: /\.css$/,
-    loaders: [
-      'style-loader',
-      ...loaderOptions
-    ]
-  },
-  production: {
-    test: /\.css$/,
-    loaders: ExtractTextPlugin.extract({
-      fallbackLoader: 'style-loader',
-      loader: [
-        ...loaderOptions
+  development: [
+    {
+      test: /\.css$/,
+      use: [
+        style,
+        ...cssOptions
       ]
-    })
-  }
+    }, {
+      test: /\.scss$/,
+      use: [
+        style,
+        ...sassOptions
```

Comment:
yes, i misread that, thought that `test` was another key. Still not totally used to this webpack syntax...